css/css-backgrounds/border-image-repeat-space-8.html

```diff
@@ -5,6 +5,7 @@
 <link rel="help" href="https://www.w3.org/TR/css3-background/#the-border-image-repeat">
 <link rel="match" href="border-image-repeat-space-8-ref.html">
 <meta name="assert" content="An explicit 'border-image-width' which is exactly the same size as 'border-width' should render the same as 'border-image-width:1' (the initial value).">
+<meta name="fuzzy" content="maxDifference=110; totalPixels=10540" />
```

Comment:
This one should have `0-` in both ranges.

Reply:
Will do!